deploy/dockerephemeral/docker-compose.yaml

```diff
@@ -161,7 +161,7 @@ services:
   elasticsearch:
     container_name: demo_wire_elasticsearch
     #image: elasticsearch:5.6
-    image: julialongtin/elasticsearch:0.0.9-amd64
+    image: elasticsearch:6.8.23
```

Comment:
unrelated, but fine.  :)

Reply:
good point, unrelated, removed from this PR.